lib/runtime-core/src/backing.rs

```diff
@@ -230,7 +230,12 @@ impl LocalBacking {
                                 }
                             };
 
-                            elements[init_base + i] = vm::Anyfunc { func, ctx, sig_id };
+                            elements[init_base + i] = vm::Anyfunc {
+                                func,
+                                ctx,
+                                sig_id,
+                                func_index: Some(func_index),
```

Comment:
Can you remove the func_index here? None of the other compilers expect an additional field.